modules/afprog/afprog.c

```diff
@@ -258,6 +258,14 @@ afprogram_sd_exit(pid_t pid, int status, gpointer s)
     }
 }
 
+static void
+afprogram_sd_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_options)
+{
+  log_msg_set_value_to_string(msg, LM_V_TRANSPORT, "local+program");
+  // log_msg_set_value(msg, LM_V_CMDLINE, self->process_info.cmdline->str, self->process_info.cmdline->len);
```

Comment:
where does this come from? do we still need it?

Reply:
I was pondering to add the command line here, but then I've figured it might actually be too much, so I didn't. this should be removed.

Reply:
fixed, thanks.